tests/tests/bitasset_tests.cpp

```diff
@@ -928,26 +928,22 @@ BOOST_AUTO_TEST_CASE( hf_935_test )
    generate_blocks( db.get_dynamic_global_properties().next_maintenance_time, true, skip );
    generate_block( skip );
 
-   for( int i = 0; i < 8; ++i )
+   for( int i = 0; i < 6; ++i )
    {
       idump( (i) );
       int blocks = 0;
       auto mi = db.get_global_properties().parameters.maintenance_interval;
 
-      if( i == 2 ) // go beyond hard fork 890
+      if( i == 2 ) // go beyond hard fork 890, 343, 935
```

Comment:
This change is not correct, because in testnet hf935 and hf890 are not at the same time.

Reply:
they are actually at the same time that's why the changes:

https://github.com/bitshares/bitshares-core/blob/master/libraries/chain/hardfork.d/CORE_935.hf
https://github.com/bitshares/bitshares-core/blob/master/libraries/chain/hardfork.d/CORE_868_890.hf

also:

https://github.com/bitshares/bitshares-core/blob/master/libraries/chain/hardfork.d/CORE_343.hf

Reply:
Please also check `testnet` branch.